cypress/integration/contributor.spec.js

```diff
@@ -7,6 +7,7 @@ import {
   cristinIDWithoutActiveAffiliation,
```

Comment:
![image](https://user-images.githubusercontent.com/46489468/144610278-fd6fd819-4006-4c8f-a490-f2407b8c23ec.png)
liten style-fiks på spinneren

Reply:
skulle gjerne sett trefflista rammet inn

Reply:
mye luft under institusjonene
![image](https://user-images.githubusercontent.com/46489468/144610416-adadbcd1-4b73-4f57-bf03-fb6b23d090c8.png)

Reply:
![image](https://user-images.githubusercontent.com/46489468/144610491-4b45133a-848f-48c8-9acf-62a12fc7ab5e.png)
Må ha veldig bred skjerm for at knappene ikke tar to linjer

Reply:
![image](https://user-images.githubusercontent.com/46489468/144610650-b82141a5-4476-4a90-97f0-42bae6243fd4.png)
Må disse være outlined ?

Reply:
Fikset